test_scripts/AppIconsFolder/ATF_P_Remove_Old_AppIconsAmountToRemove_noSpace.lua

```diff
@@ -0,0 +1,179 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--    [GENIVI] Conditions for SDL to create and use 'AppIconsFolder' storage 
+--    [AppIconsFolder]: Conditions for SDL to remove old icons on value defined at "AppIconsAmountToRemove" param
+-- Description:
+--    SDL should remove oldest icon due AppIconsAmountToRemove param in .ini file if not enough space 
+-- 1. Used preconditions:
+--      Stop SDL
+--      Set AppIconsFolder in .ini file
+--      Set AppIconsFolder maxSize
+--      Set Icons Amount to remove if not enough space 
+--      Start SDL and HMI
+--      Connect mobile
+--      Make AppIconsFolder is full
+-- 2. Performed steps:
+--      Register app
+--      Send SetAppIcon
+-- Expected result:
+--    SDL should delete oldest icon and save a new one in AppIconsFolder  
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+
+--[[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+require('user_modules/AppTypes')
+
+--[[ Local variables ]]
+local RAIParameters = config.application1.registerAppInterfaceParams
+local testIconsFolder = commonPreconditions:GetPathToSDL() .. "IconsFolder"
```

Comment:
@aarotar, 'testIconsFolder' is unused. Please use it instead of hard coded 'IconsFolder' value